src/main.cpp

```diff
@@ -174,6 +175,38 @@ void DDWAF::updateRuleData(const Napi::CallbackInfo& info) {
   ddwaf_object_free(&data);
 }
 
+void DDWAF::toggleRules(const Napi::CallbackInfo& info) {
+  mlog("Toggling rules on DDWAF");
+  Napi::Env env = info.Env();
+  if (this->_disposed) {
+    Napi::Error::New(env, "Could not toggle rule on a disposed WAF").ThrowAsJavaScriptException();
+    return;
+  }
+  if (info.Length() < 1) {
+    Napi::Error::New(env, "Wrong number of arguments, expected 1").ThrowAsJavaScriptException();
+    return;
+  }
+  if (!info[0].IsObject()) {
+    Napi::TypeError::New(env, "First argument must be an object").ThrowAsJavaScriptException();
+    return;
+  }
+
+  ddwaf_object rulesToggleMap;
+  to_ddwaf_object(&rulesToggleMap, env, info[0], 0, false, false);
+  DDWAF_RET_CODE code = ddwaf_toggle_rules(this->_handle, &rulesToggleMap);
```

Comment:
little bit of spacing
```suggestion

  DDWAF_RET_CODE code = ddwaf_toggle_rules(this->_handle, &rulesToggleMap);
```